src/resources/js/dependency.js

```diff
@@ -72,6 +75,7 @@
 						}
 					} else {
 						$dependent
+							.hide()
```

Comment:
That was an example; the case at hand has 3 classes nested in specificity.
We would end up having to take any possible use into account to use CSS to hide show, what's the problem with JS hide/show in this specific case? It's setting inline styles.